client/components/common/ReactDayPicker/Weekday.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { WeekdayElementProps } from 'react-day-picker/';
```

Comment:
nice! looks like you figured out the props errors.

Reply:
Yeah. I got it working.